scripts/scrapper.ts

```diff
@@ -1,10 +1,44 @@
 import axios from 'axios'
 import * as cheerio from 'cheerio'
-import { MealCategory } from '#models/meal'
+import Meal, { MealCategory } from '#models/meal'
+import { createHash } from 'node:crypto'
+import db from '@adonisjs/lucid/services/db'
+import WebsiteHash from '#models/website_hash'
+import logger from '@adonisjs/core/services/logger'
 
-export async function scrapeMenu() {
-  const url = 'https://sks.pwr.edu.pl/menu/'
+const url = 'https://sks.pwr.edu.pl/menu/'
+
+export async function runScrapper() {
+  const currentHash = await cacheMenu()
+  const storedHash = await WebsiteHash.first()
+
+  const trx = await db.transaction()
+  try {
+    if (!storedHash) {
+      logger.info('No hash found in database. Storing current hash and scraping menu.')
+      await WebsiteHash.create({ hash: currentHash }, { client: trx })
+    } else if (storedHash.hash === currentHash) {
+      logger.info('Did not find any differences. Not proceeding with scraping.')
+      return
+    }
+
+    const meals = await scrapeMenu()
+    await Meal.query({ client: trx }).delete()
```

Comment:
didn't Linek want to store them to have history and stats?

Reply:
didn't we finally agree that checking correlation of fries price to students' grades is not a feature that we want?

but i'm not sure (should have taken notes)

Reply:
not fully sure too, but taking his ambitions into account I wouldn't be so sure

Reply:
assuming we want the history, you can store multiple hashes i the hash table with stamps and add relationship between hash and menu items

Reply:
what do u think about leaving this one for the next feature?

i can release the basic one pretty soon and work on this one later

Reply:
Sure, this can be done later

Reply:
Created an issue for this one